sdks/csharp/twin_operations_sample.cs

```diff
@@ -77,36 +75,38 @@ public static async Task Main(string[] args)
             await DeleteTwin(client, twin_ID);
         }
 
-        private static BasicDigitalTwin FetchAndPrintTwin(string twin_ID, DigitalTwinsClient client)
+        private static async Task<BasicDigitalTwin> FetchAndPrintTwinAsync(string twinId, DigitalTwinsClient client)
         {
-            BasicDigitalTwin twin;
             // <GetTwin>
+            BasicDigitalTwin twin;
             // <GetTwinCall>
-            Response<BasicDigitalTwin> twin = client.GetDigitalTwin(twin_ID);
+            Response<BasicDigitalTwin> twinResponse = await client.GetDigitalTwinAsync(twinId);
```

Comment:
We should always show the async flavors of these APIs because we prefer people to use async. If they use the synchronous variety, it will eventually go async because HttpClient.Send is async.

If their application is async, and they call a sync method, which in turns calls an async method, it could end in a deadlock. It is a bad pattern.